src/node_file.cc

```diff
@@ -479,9 +479,9 @@ void FillStatsArray(double* fields, const uv_stat_t* s) {
   fields[9] = -1;
 #endif
   // Dates.
-#define X(idx, name)                                                          \
-  fields[idx] = (static_cast<double>(s->st_##name.tv_sec) * 1000) +           \
-                (static_cast<double>(s->st_##name.tv_nsec / 1000000));        \
+#define X(idx, name)                          \
+  fields[idx] = (s->st_##name.tv_sec * 1e3) + \
+                (s->st_##name.tv_nsec / 1e6); \
```

Comment:
Ohh yeah, I didn't follow the braces, assumed it was the same as the line above...